cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -107,14 +109,16 @@
     UPGRADE("Upgrades CDAP to " + ProjectInfo.getVersion() + "\n" +
               "  The upgrade tool upgrades the following: \n" +
               "  1. User and System Datasets (upgrades the coprocessor jars)\n" +
-              "  2. UsageRegistry Dataset Type\n" +
-              "  3. Application Specifications\n" +
-              "  4. Removes all Adapters\n" +
-              "  5. Application Specifications\n" +
+              "  2. Application Specifications\n" +
+              "  3. Removes all Adapters\n" +
+              "  4. Application Specifications\n" +
               "      - Adds artifacts for existing applications\n" +
               "      - Updates application metadata to include newly added artifact\n" +
```

Comment:
No, I realized you didn't change it, but I figured these are always a good chance to review things...the change depends on how precise we'd like to be. Given that there is one artifact referenced in each application metadata, it could be: 
"Updates each application's metadata to include the newly added artifact", to make clear the relationship...